frontend/src/employee-frontend/components/child-information/assistance-need/decision/AssistanceNeededDecisionForm.tsx

```diff
@@ -68,21 +69,41 @@ const formatName = (someone: {
   lastName: string | null
 }) => `${someone.lastName ?? ''} ${someone.firstName ?? ''}`
 
-const EmployeeSelectWithTitle: React.FC<{
+interface EmployeeSelectProps {
   employeeLabel: string
   employees: Employee[]
   selectedEmployee: Employee | null
   titleLabel: string
   selectedTitle: string | null
+  phoneLabel?: undefined
+  selectedPhone?: undefined
+  onChangePhone?: undefined
   onChange: (employee: Employee | null, title: string | null) => void
   'data-qa'?: string
-}> = ({
+}
+
+interface EmployeeSelectWithPhoneProps
+  extends Omit<
+    EmployeeSelectProps,
+    'phoneLabel' | 'selectedPhone' | 'onChangePhone'
+  > {
+  phoneLabel: string
+  selectedPhone: string
+  onChangePhone: (phone: string) => void
+}
+
+const EmployeeSelectWithTitle: React.FC<
+  EmployeeSelectProps | EmployeeSelectWithPhoneProps
```

Comment:
As the `EmployeeSelectWithTitle` operates on a data structure that always looks the same (has a `phoneNumber` field), I'd probably have just extended the props to take this into account and added a `hidePhoneNumberField` (or similar) prop.

In this case the phone number could also be added to the `onChange` function, which could simplify the usage points as well.

I think something like that would be a bit simpler. What do you think?